app/views/api/v3/statements/show.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+json.id @statement.api_id
+json.type "statement"
+
+json.attributes do
+  json.month Date::MONTHNAMES[@statement.month]
```

Comment:
How would we handle different serialisations for the versions of the API with jbuilder? it looks like its scoped to the API version here, so would sharing/extending for other versions be a pain?